articles/tutorials/building_2d_games/index.md

```diff
@@ -0,0 +1,74 @@
+
+| Appendix | Summary |
+| -------- | ------- |
+|          |         |
+
+## Conventions Used in This Documentation
+
+The following conventions are used in this documentation
+
+### Italics
+
+*Italics* are used for emphasis, technical terms, and paths such as file paths including filenames and extensions.
+
+### Inline Code Blocks
+
+`Inline code` blocks are used for methods, functions, and variable names when they are discussed with the body a text paragraph.
+
+### Code Blocks
+
+```cs
+// Example Code Block
+public void Foo() { }
+```
+
+Code blocks are used to show code examples with syntax highlighting
+
+## MonoGame
+
+If you ever have questions about MonoGame or would like to talk with other developers to share ideas or just hang out with us, you can find us in the various MonoGame communities below
+
+* [Discord](https://discord.gg/monogame)
+* [GitHub Discussions Forum](https://github.com/MonoGame/MonoGame/discussions)
+* [Community Forums (deprecated)](https://community.monogame.net/)
+* [Reddit](https://www.reddit.com/r/monogame/)
+* [Facebook](https://www.facebook.com/monogamecommunity)
+
+## Note From Author
+
+> I have been using MonoGame for the past several years (since 2017). It was a time in my game development journey where I was looking for something that I had more control over. I didn't want to spend the time to write a full game engine, but I also wanted to have more control than what the current engines at the time (i.e. Unity) offered. At that time, there was a vast amount of resources available for getting started, but none of them felt like they fit a good beginner series. Even now, the resources available still seem this way. They either require the reader to have a great understanding of game development and programming, or they assume the reader has none and instead focuses on teaching programming more than teaching MonoGame. Even still, some relied too heavily on third party libraries, others were simply very bare bones asking the reader to just copy and paste code without explaining the *what* of it all.
+> 
+> Since then, I have written various small self contained tutorials on different topics for MonoGame to try and give back to the community for those getting started. I also participate regularly in the community discussion channels, answering questions and offering technical advice, so I'm very familiar with the topics and pain points that users get hung up on when first starting out.
+> 
+> My hopes with this documentation is to take the lessons I've learned, and what I've learned from helping others, to fill the gap that I wanted filled when I first started MonoGame. To present using MonoGame in a straight forward way, introducing concepts and building off of them as we go along in a way that makes sense and is easy to follow.

```

Comment:
Not fond of this sentence hard to read and round about way of saying something simple.
Alternative:
With this documentation I hope to take all the lessons I've learnt and provide a tutorial series that I wish was available when I first started.

Reply:
Thanks @MutsiMutsi, I've adjusted the wording based on your recommendation.